objects.md

```diff
@@ -3,125 +3,199 @@ Object Model
 
 ### Export object
 
-  Key          Value Type   Value Description
-  ------------ ------------ -----------------------------------------------------------
-  datatype     string       The datatype (e.g., LAS 1.2, DTM).
-  hsrs         string       The Horizontal Spatial Reference System EPSG code.
-  name         string       The name of the export.
-  pk           integer      The primary key of the export.
-  started\_at   timestamp    Time of creation for the AOI: `YYYY-MM-DD HH24:MI:SS.FF6`
-  status       string       The status of the export (e.g., SUCCESS, FAILED, QUEUED).
-  url          string       The download URL of the export.
+  Key                 Value Type   Value Description
+  ------------------- ------------ -----------------------------------------------------------
+  datatype            string       The datatype (e.g., LAS 1.2, DTM).
+  user                integer      The id of the creating user.
+  hsrs                string       The Horizontal Spatial Reference System EPSG code.
+  name                string       The name of the export.
+  pk                  integer      The primary key of the export.
+  started\_at         timestamp    Time of creation for the AOI. ISO 8601 format as UTC.
+  status              string       The status of the export (e.g., SUCCESS, FAILED, QUEUED).
+  url                 string       The download URL of the export.
+
+### Export Detail object (pointcloud)
+
+  ----------------------------------------------------------------------------------------------------------------------------------
+  Key                   Value Type                                              Value Description
+  --------------------- ------------------------------------------------------- ----------------------------------------------------
+  datatype              string                                                  The datatype (e.g., LAS 1.2, DTM).
+  
+  user                  integer                                                 The id of the creating user.
+  
+  hsrs                  string                                                  The Horizontal Spatial Reference System EPSG code.
+  
+  name                  string                                                  The name of the export.
+  
+  pk                    integer                                                 The primary key of the export.
+  
+  started\_at           timestamp                                               Time of creation for the AOI. ISO 8601 format as UTC.
+  
+  status                string                                                  The status of the export (e.g., SUCCESS, FAILED, QUEUED).
+  
+  url                   string                                                  The download URL of the export.
+  
+  rgb                   boolean                                                 Whether or not RGB dimension is included in exported data.
```

Comment:
This is a good idea.  I think our classification reporting in the UI could also benefit from an overhaul.  This will have to be targeted in the next release though.